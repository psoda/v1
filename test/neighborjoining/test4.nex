#NEXUS


begin data;
	dimensions ntax=6 nchar=29;
	format datatype=dna gap=- matchchar=. ;
matrix

Species1  ACTGACTGACTGACTGACTGACTGACTGA
Species2  ACTGACTGACTGACTGACTGACTGACTGA
Species3  ACTGACTGACTGACTGACTGACTGACTGA
Species4  ACTGACTGACTGACTGACTGACTGACTGA
Species5  ACTGACTGACTGACTGACTGACTGACTGA
Species6  ACTGACTGACTGACTGACTGACTGACTGA
;
end;
BEGIN PAUP;
	set nowarnreset 
	autoclose;
	hsearch start=nj swap=none;
    pscores;
[This block will estimate a maximum parsimony tree assuming equal weights and save the results of the search to a log file.]
  quit warntsave=no;
end;
