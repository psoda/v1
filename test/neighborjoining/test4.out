Swap NONE, not doing main search
## HSearch Completed Successfully
score 0.000000
## PScores Completed Successfully

Done. (at "test4.nex" Line 23:3--23:20)
